components/search/SearchInterface.vue

```diff
@@ -116,9 +116,9 @@
 
 <script>
   import AlertMessage from '../../components/generic/AlertMessage';
+  import SearchResults from '../../components/search/SearchResults';
```

Comment:
To save future confusion, let's add an inline comment here: `...; // Sorted before InfoMessage to prevent Conflicting CSS sorting warning`

Reply:
Done